### ISSUE TITLE
Persist token

### ISSUE THREAD

**Issue by jvdutra** [orkestral/venom#57]
I'm currently using Heroku and everytime my Dyno restarts I need to rescan the code. Is there a way to persist the token to a database, and not to a file?

Thanks. Amazing work.

**Comment by hbdbim** [comment]
it currently doesn't, but you can implement it by passing the session in the create method.

**Comment by Dhyeythumar** [comment]
@jvdutra how did you persist the tokens & use them after restart? Because for me it's not working.

I have tried to do this with the help of `env variables/config vars`, below are the steps that I followed:
- I start the worker dyno on Heroku & will get all the tokens with the help of `await Client.getSessionTokenBrowser()` 
- After this, I will set the config vars using these tokens,
- Now because there was a change in config vars the dyno gets restarted
- But after the restart, it again asks me to scan, Why?

So if it's manageable to persist the tokens, then why it's not accepting the tokens for the next run? 

**Is this happening because the dyno restarted?** _(does that mean the environment got changed and a new browser is opened & now the tokens are not valid.)_
